src/journal/journald-context.c

```diff
@@ -268,6 +272,40 @@ static int client_context_read_label(
         return 0;
 }
 
+static int client_context_read_log_filter_regex(ClientContext *c, const char *cgroup) {
+        _cleanup_free_ char *pattern = NULL;
+        char *dummy;
+        pcre2_code *new_regex = NULL;
+        bool deny_list = false;
+        int r;
+
+        assert(c);
+        assert(cgroup);
+
+        r = cg_get_xattr_malloc(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.journald_log_filter_regex",
+                                &pattern);
+        if (r == -ENODATA)
+                /* -ENODATA is returned when the key is not defined, this is not an error. */
+                return 0;
+        if (r < 0)
+                return log_debug_errno(r, "Failed to get user.journald_log_filter_regex xattr for %s: %m",
+                                       cgroup);
+
+        dummy = pattern;
+        if (dummy[0] == '~') {
+                deny_list = true;
+                ++dummy;
```

Comment:
We prefer `dummy++` I think

Reply:
Fixed. Any specific reasoning behind this?

Reply:
Nope, it's just what's used everywhere else